templates/base.html

```diff
@@ -48,6 +48,12 @@
         integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous">
     </script>
 {% block js %}
+<script>
+// Fires toast popups if there are messages
+{% if messages %}

```

Comment:
What is this needed for?

Reply:
This is from @Ri-Dearg but I am guessing this is to fire toast alert when a message happens in Django. It's explained here: https://pypi.org/project/django-toast-messages/. We need to insert a script in the middle of a django template tag